examples/jsm/loaders/PLYLoader.js

```diff
@@ -363,25 +363,51 @@ class PLYLoader extends Loader {
 
 		function handleElement( buffer, elementName, element ) {
 
+			function findAttrName( names ) {
+
+				for ( let i = 0, l = names.length; i < l; i ++ ) {
```

Comment:
I've notice that a major part of the performance drop was introduced by the for/in loop. Using a canonical one makes it much faster.

Reply:
Things never change... 😁

I suspect it's okay that is a bit slower.